app/views/shared/_asset_table.html.slim

```diff
@@ -0,0 +1,38 @@
+.float-left
+  table
+    thead
+      tr
+        th Item model
+        th Item description
+        th Item price
+        th Item attributes
+        th Category
+        th Assignee
+        th
+
+    tbody
+      - @assets.each do |asset|
+        tr
+          td = asset.item_model
+          td = asset.item_description
+          td = asset.item_price
+          td = asset.item_attributes
+          td = asset.category
+          td = asset.assignee
+          td 
+            = link_to 'Show', asset, class: 'btn btn-primary'
+            = link_to 'Edit', edit_asset_path(asset), class: 'btn btn-primary'
+            = link_to 'Delete', asset, data: {:confirm => 'Are you sure?'}, :method => :delete, class: 'btn btn-primary'
+
+  br
+
+  = link_to 'New Asset directory', new_asset_path, class: 'btn btn-primary'
+
+.float-right
+  h2 Search Bar
+  form.search-form
+    | Enter Search item here:
+    br
+    input name="username" type="text" 
+    br
+    input.button onclick="alert('Search')" type="button" value="Search" 
```

Comment:
I just slacked you the user preferences you must put into your sublime.

The line: "ensure_newline_at_eof_on_save": true,
Will make sure that you don't get that little red arrow this at the end of a file when viewing on github.

So go back to this file after you've updated your user preferences and then save this file and it should create the new line on the bottom of the file automatically.